definitions/private-law/json/AuthorisationCaseField/SendToGatekeeper/AuthorisationCaseField.json

```diff
@@ -83,6 +83,48 @@
     "UserRole": "caseworker-privatelaw-judge",
     "CRUD": "CRUD"
   },
+  {
+    "LiveFrom": "25/01/2023",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "letGateKeepersKnowLabel",
+    "UserRole": "caseworker-privatelaw-la",
+    "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "25/01/2023",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "sendToGateKeeperHint",
+    "UserRole": "caseworker-privatelaw-la",
+    "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "25/01/2023",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "isSpecificGateKeeperNeeded",
+    "UserRole": "caseworker-privatelaw-la",
+    "CRUD": "CRUD"
```

Comment:
We need CRU. CRUD dont think we need for any. We should also use access profile rather than adding individual role wise permission.